packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -0,0 +1,287 @@
+<template>
+    <li
+        :class="['c-countrySelector c-nav-list-item has-sublist', {
+            'is-open': countrySelectorIsOpen
+        }]"
+        data-test-id="countrySelector-toggle"
+        v-on="isBelowMid ? null : { mouseover: openCountrySelector, mouseleave: closeCountrySelector }"
+        @keyup.esc="closeCountrySelector">
+        <f-button
+            :tabindex="isBelowMid && !navIsOpen ? -1 : 0"
+            button-type="icon"
+            class="c-nav-list-text c-countrySelector-btn"
+            :aria-expanded="!isBelowMid && countrySelectorIsOpen ? 'true' : 'false'"
+            :aria-haspopup="isBelowMid ? false : true"
+            :aria-label="copy.changeCurrentCountry"
+            @click="onCountrySelectorToggle"
+            v-on="isBelowMid && !countrySelectorIsOpen ? { blur: closeNav, focus: openNav } : null">
+            <span class="c-countrySelector-currentFlag-wrapper">
+                <flag-icon
+                    :country-code="copy.currentCountryKey"
+                    class="c-countrySelector-flag c-countrySelector-flag--current" />
+            </span>
+            <span class='c-countrySelector-title'>
+                {{ copy.selectYourCountryText }}
+            </span>
+        </f-button>
+
+        <div :class="['c-nav-panel', { 'is-visible': countrySelectorIsOpen }]">
+            <div class="c-nav-popoverList c-nav-popoverList--countrySelector">
+                <header class="c-nav-popoverList-header">
+                    <f-button
+                        button-type="icon"
+                        button-size="xsmall"
+                        class="c-nav-popoverList-header-button"
+                        aria-label="Go back to main menu"
```

Comment:
Do we have copy for this for all tenants?

Reply:
Oh, actually we don't have translations for this. I will add it in english for all the tenants for now and will ask for translations